shipper.py

```diff
@@ -126,13 +125,13 @@ def shipment_new():
                     'user': user_entitled,
                     'status': 'new',
                     'action': request.forms.get('action'),
-                    'md': new_md
+                    'md': new_md,
+                    'file_id': request.forms.get('file_id')
```

Comment:
`file_id` is not documented yet in https://github.com/o2r-project/o2r-web-api/pull/41 - what does it do?
(same for `action` afaics)